### DISCUSSION TITLE
Hiding a whole beatmapset results in "no beatmaps available" as the current carousel beatmap

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#30468] [Q&A]
Not sure if this was intended.

https://github.com/user-attachments/assets/cb17cb2a-f116-4ece-bbc9-0e3ebc7fa66a

**Comment by BunnyTub** [comment]
Wouldn't this be an issue?

**Reply by Gabixel** [reply]
I guess so. Wasn't really sure at first (for.. some odd reason).

**Reply by mixalisnew** [reply]
If you hide it how you unhide it?

**Reply by Gabixel** [reply]
There's a dedicated option near the end of the settings panel to show all hidden difficulties. I think you can search for "hidden". No other way
